### PR TITLE
Better using facing string

### PR DESCRIPTION
**Pull request by wli3**
Fix https://github.com/dotnet/cli/issues/8728

Fix https://github.com/dotnet/cli/issues/8369

### CODE REVIEW

File: src/dotnet/commands/dotnet-install/dotnet-install-tool/xlf/LocalizableStrings.cs.xlf [hunk 1]
```diff
@@ -105,18 +105,18 @@ Instalace byla úspěšná. Pokud nejsou další pokyny, můžete přímo do já
...
-        <source>Cannot have global and tool-path as opinion at the same time.</source>
-        <target state="new">Cannot have global and tool-path as opinion at the same time.</target>
+        <source>(--global) conflicts with the tool path option (--tool-path). Please specify only one of the options.</source>
+        <target state="new">(--global) conflicts with the tool path option (--tool-path). Please specify only one of the options.</target>
         <note />
       </trans-unit>
       <trans-unit id="ToolPathDescription">
-        <source>Location of shim to access tool</source>
-        <target state="new">Location of shim to access tool</target>
+        <source>LLocation where the tool will be installed.</source>
```
Comment (KathleenDollard) [inline src/dotnet/commands/dotnet-install/dotnet-install-tool/xlf/LocalizableStrings.cs.xlf]: Typo

File: src/dotnet/commands/dotnet-install/dotnet-install-tool/xlf/LocalizableStrings.de.xlf [hunk 2]
```diff
@@ -105,18 +105,18 @@ Die Installation war erfolgreich. Sofern keine weiteren Anweisungen vorliegen, k
...
-        <target state="new">Cannot have global and tool-path as opinion at the same time.</target>
+        <source>(--global) conflicts with the tool path option (--tool-path). Please specify only one of the options.</source>
+        <target state="new">(--global) conflicts with the tool path option (--tool-path). Please specify only one of the options.</target>
         <note />
       </trans-unit>
       <trans-unit id="ToolPathDescription">
-        <source>Location of shim to access tool</source>
-        <target state="new">Location of shim to access tool</target>
+        <source>LLocation where the tool will be installed.</source>
+        <target state="new">LLocation where the tool will be installed.</target>
```
Comment (KathleenDollard) [inline src/dotnet/commands/dotnet-install/dotnet-install-tool/xlf/LocalizableStrings.de.xlf]: Typo LL

File: src/dotnet/commands/dotnet-install/dotnet-install-tool/xlf/LocalizableStrings.es.xlf [hunk 3]
```diff
@@ -105,18 +105,18 @@ La instalación se completó correctamente. Si no hay más instrucciones, puede
...
-        <target state="new">Cannot have global and tool-path as opinion at the same time.</target>
+        <source>(--global) conflicts with the tool path option (--tool-path). Please specify only one of the options.</source>
+        <target state="new">(--global) conflicts with the tool path option (--tool-path). Please specify only one of the options.</target>
         <note />
       </trans-unit>
       <trans-unit id="ToolPathDescription">
-        <source>Location of shim to access tool</source>
-        <target state="new">Location of shim to access tool</target>
+        <source>LLocation where the tool will be installed.</source>
+        <target state="new">LLocation where the tool will be installed.</target>
```
Comment (KathleenDollard) [inline src/dotnet/commands/dotnet-install/dotnet-install-tool/xlf/LocalizableStrings.es.xlf]: Typo

File: src/dotnet/commands/dotnet-install/dotnet-install-tool/xlf/LocalizableStrings.fr.xlf [hunk 4]
```diff
@@ -105,18 +105,18 @@ L'installation a réussi. En l'absence d'instructions supplémentaires, vous pou
...
-        <target state="new">Cannot have global and tool-path as opinion at the same time.</target>
+        <source>(--global) conflicts with the tool path option (--tool-path). Please specify only one of the options.</source>
+        <target state="new">(--global) conflicts with the tool path option (--tool-path). Please specify only one of the options.</target>
         <note />
       </trans-unit>
       <trans-unit id="ToolPathDescription">
-        <source>Location of shim to access tool</source>
-        <target state="new">Location of shim to access tool</target>
+        <source>LLocation where the tool will be installed.</source>
+        <target state="new">LLocation where the tool will be installed.</target>
```
Comment (KathleenDollard) [inline src/dotnet/commands/dotnet-install/dotnet-install-tool/xlf/LocalizableStrings.fr.xlf]: Typo

File: src/dotnet/commands/dotnet-install/dotnet-install-tool/xlf/LocalizableStrings.it.xlf [hunk 5]
```diff
@@ -105,18 +105,18 @@ L'installazione è riuscita. Se non ci sono altre istruzioni, è possibile digit
...
-        <source>Cannot have global and tool-path as opinion at the same time.</source>
-        <target state="new">Cannot have global and tool-path as opinion at the same time.</target>
+        <source>(--global) conflicts with the tool path option (--tool-path). Please specify only one of the options.</source>
+        <target state="new">(--global) conflicts with the tool path option (--tool-path). Please specify only one of the options.</target>
         <note />
       </trans-unit>
       <trans-unit id="ToolPathDescription">
-        <source>Location of shim to access tool</source>
-        <target state="new">Location of shim to access tool</target>
+        <source>LLocation where the tool will be installed.</source>
```
Comment (KathleenDollard) [inline src/dotnet/commands/dotnet-install/dotnet-install-tool/xlf/LocalizableStrings.it.xlf]: Typo, just search for the rest :)

File: src/dotnet/commands/dotnet-install/dotnet-install-tool/xlf/LocalizableStrings.cs.xlf (hunk 1 above)
Comment (wli3) [reply src/dotnet/commands/dotnet-install/dotnet-install-tool/xlf/LocalizableStrings.cs.xlf]: Sorry, correcting

Comment (wli3) [reply src/dotnet/commands/dotnet-install/dotnet-install-tool/xlf/LocalizableStrings.cs.xlf]: Fixed

**Comment by wli3** [comment]
@dotnet/dotnet-cli need an approval of this PR

File: src/dotnet/commands/dotnet-uninstall/tool/LocalizableStrings.resx [hunk 6]
```diff
@@ -145,12 +145,12 @@
     <value>Failed to uninstall tool '{0}': {1}</value>
   </data>
   <data name="UninstallToolCommandNeedGlobalOrToolPath" xml:space="preserve">
-    <value>Need either global or tool-path provided.</value>
+    <value>Please specify either the global option (--global) or the tool path option (--tool-path).</value>
   </data>
   <data name="ToolPathDescription" xml:space="preserve">
-    <value>Location of shim to access tool</value>
+    <value>Location where the tool will be installed.</value>
```
Comment (peterhuene) [inline src/dotnet/commands/dotnet-uninstall/tool/LocalizableStrings.resx]: This should be `Location where the tool was previously installed.`

Comment (wli3) [reply src/dotnet/commands/dotnet-uninstall/tool/LocalizableStrings.resx]: Changed as suggested

**Comment by peterhuene** [comment]
👍

**Pull request merged**